Human/nbconverted/Human_experiment_lvl_sim.py

```diff
@@ -1,4 +1,4 @@
-
+#!/usr/bin/env python
```

Comment:
This isn't a criticism so much as something I'm curious about: why did you decide to add a shebang line here? I don't see it very frequently for python scripts, though I imagine there's a reason for you adding it

Reply:
Is it something that nbconvert started doing?

Reply:
I hadn't noticed that before. It does look like its something that is added when you nbconvert: https://github.com/jupyter/nbconvert/issues/690